src/util/schema_keywords.js

```diff
@@ -0,0 +1,165 @@
+/* Copyright (C) 2016 NooBaa */
+'use strict';
+
+const js_utils = require('./js_utils');
+const { _: codegen} = require('ajv');
+
+/**
+ * @typedef {import('ajv').KeywordCxt} KeywordCxt
+ */
+
+const KEYWORDS = js_utils.deep_freeze({
+
+    // schema: { date: true } will match (new Date())
+    date: {
+        keyword: 'date',
```

Comment:
You can add schemaType when possible, in this case:
```
  schemaType: 'boolean',
```

Reply:
It then prompts a ts error.
So for now it is commeneted

Reply:
OK, I wonder where this errors occur - in which api/schema we use the date keyword other than `date: true` ...